<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XBee Configuration Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>false</bool>
      </property>
      <widget class="TextConsole" name="console">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;_&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="destinationLabel">
            <property name="text">
             <string>Destination Address</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="destinationAddress">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>FFFF</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="networkLabel">
            <property name="text">
             <string>Network Address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="networkAddress">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>7FFF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="commandList"/>
        </item>
        <item>
         <widget class="QLabel" name="comment">
          <property name="text">
           <string>Comments</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="remoteConfigCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remote Configure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="apiCheckBox">
            <property name="text">
             <string>API Enable</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="readButton">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="writeButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>&amp;Configuration</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionDigi_Homepage"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionClear_Console"/>
   </widget>
   <addaction name="menuConfiguration"/>
   <addaction name="menuShow"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionClear_Console"/>
   <addaction name="actionFull_Screen"/>
   <addaction name="actionAbout"/>
   <addaction name="actionDigi_Homepage"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/messagebox_info.png</normaloff>:/icons/icons/messagebox_info.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/application_exit.png</normaloff>:/icons/icons/application_exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/connect_established.png</normaloff>:/icons/icons/connect_established.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/document_properties.png</normaloff>:/icons/icons/document_properties.png</iconset>
   </property>
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/connect_no.png</normaloff>:/icons/icons/connect_no.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionClear_Console">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/clear_left.png</normaloff>:/icons/icons/clear_left.png</iconset>
   </property>
   <property name="text">
    <string>Clear Console</string>
   </property>
  </action>
  <action name="actionDigi_Homepage">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/konqueror.png</normaloff>:/icons/icons/konqueror.png</iconset>
   </property>
   <property name="text">
    <string>Digi Webpage</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/window_fullscreen.png</normaloff>:/icons/icons/window_fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextConsole</class>
   <extends>QTextEdit</extends>
   <header>textconsole.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>apiCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>destinationAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apiCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>networkAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apiCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>remoteConfigCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
